modules/ocf_hpc/manifests/compute.pp

```diff
@@ -1,6 +1,11 @@
 class ocf_hpc::compute {
   require ocf_hpc
 
+  include ocf::ipmi
+  include ocf::firewall::allow_ssh
+
```

Comment:
style: any reason for the space?

Reply:
Thought it looked slightly better to separate the ocf_hpc subclass from the others (though I didn't do it for the other one lol)